data-pipeline-flink/telemetry-extractor/src/main/scala/org/sunbird/dp/task/ExtractorStreamTask.scala

```diff
@@ -0,0 +1,97 @@
+package org.sunbird.dp.task
+
+import java.util
+
+import org.apache.flink.api.common.typeinfo.TypeInformation
+import org.apache.flink.api.java.typeutils.TypeExtractor
+import org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment
+import org.apache.flink.streaming.api.scala.OutputTag
+import org.sunbird.dp.functions.{DeduplicationFunction, ExtractionFunction}
+
+
+class ExtractorStreamTask(config: ExtractionConfig) extends BaseStreamTask(config) {
+
+  private val serialVersionUID = -7729362727131516112L
+
+  def process(): Unit = {
+    implicit val env: StreamExecutionEnvironment = StreamExecutionEnvironment.getExecutionEnvironment
+    implicit val eventTypeInfo: TypeInformation[util.Map[String, AnyRef]] = TypeExtractor.getForClass(classOf[util.Map[String, AnyRef]])
```

Comment:
Is the implicit declaration required?

Reply:
Yes Santhosh .. We need to declare this implicit value or we can import **org.apache.flink.api.scala._** this package.